### PR TITLE
electrolysis oversight fix 

### LINKED ISSUE #77121: Etherial blood does not react with water in "Electrolysis" reaction
Reporting client version: 514.1589

## Round ID:
211058

### PR DESCRIPTION
**Pull request by Vincent983**
## About The Pull Request

fixes https://github.com/tgstation/tgstation/issues/77121 , making the electrolysis reaction work with normal ethereal blood as well as the food version.

## Why It's Good For The Game

I think it's a cute reaction to have, and it makes more sense to have both versions of liquid electricity work for electrolysis.
(powercreep)
## Changelog

:cl:
fix: normal ethereal blood now works for electrolysis, the hydrogen and oxygen output of the electrolysis recipe has been increased.
/:cl:

### CODE REVIEW

**Comment by Vincent983** [comment]
wait hol on

**Comment by Vincent983** [comment]
there, now both reagents work for the reaction.

**Comment by AtomTheProphet** [comment]
I like this. this is good.

**Comment by HalfPunch** [comment]
Etherial ghetto chemists let's goo!👍

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 1]
```diff
@@ -592,9 +592,13 @@
 //water electrolysis
 /datum/chemical_reaction/electrolysis
 	results = list(/datum/reagent/oxygen = 1.5, /datum/reagent/hydrogen = 3)
```
Comment (Pickle-Coding) [inline code/modules/reagents/chemistry/recipes/others.dm:594]: Electrolysing 5 moles (units = moles and are treated as such) of water would result it 5 moles of hydrogen (we assume hydrogen is H2) and 2.5 moles of oxygen (we assume oxygen is O2).

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 2]
```diff
@@ -592,9 +592,13 @@
...
-	required_reagents = list(/datum/reagent/consumable/liquidelectricity/enriched = 1, /datum/reagent/water = 5)
+	required_reagents = list(/datum/reagent/consumable/liquidelectricity = 1, /datum/reagent/water = 5)
 	reaction_tags = REACTION_TAG_EASY | REACTION_TAG_CHEMICAL
 
+/datum/chemical_reaction/electrolysis2
+	results = list(/datum/reagent/oxygen = 1.5, /datum/reagent/hydrogen = 3)
```
Comment (Pickle-Coding) [inline code/modules/reagents/chemistry/recipes/others.dm:599]: Ditto.

File: code/modules/reagents/chemistry/recipes/others.dm (hunk 1 above)
Comment (Vincent983) [reply code/modules/reagents/chemistry/recipes/others.dm:594]: okie, I just copied the existing electrolysis reaction

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 3]
```diff
@@ -591,10 +591,14 @@
 
 //water electrolysis
 /datum/chemical_reaction/electrolysis
-	results = list(/datum/reagent/oxygen = 1.5, /datum/reagent/hydrogen = 3)
-	required_reagents = list(/datum/reagent/consumable/liquidelectricity/enriched = 1, /datum/reagent/water = 5)
+	results = list(/datum/reagent/oxygen = 2.5, /datum/reagent/hydrogen = 5)
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/recipes/others.dm:594]: should mention this in the changelog

**Pull request merged**